New package: PartitionedLS v0.9.5

JuliaRegistrator opened the pull request:
- Registering package: PartitionedLS
- Repository: https://github.com/ml-unito/PartitionedLS.jl
- Created by: @boborbt
- Version: v0.9.5
- Commit: 5f3df31c8c9f6db338c6a373a1a8515099422a30
- Git reference: HEAD
- Description: Implements the Partitioned Least Squares algorithms

boborbt commented:
It seems that all checks are good. This is my first time here, I have no idea how to investigate what the problem is...

JuliaHubRegistrator committed:
New package: PartitionedLS v0.9.5

UUID: 19f41c5e-8610-11e9-2f2a-0d67e7c5027f
Repo: https://github.com/ml-unito/PartitionedLS.jl.git
Tree: 9931f94ed31610fc072b7b1a21e354284765f338

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

boborbt commented:
> Welcome!

Thank you

> > It seems that all checks are good
> 
> What do you mean by that?

In the section showing the automated checks, apparently all had passed, but the last one, which just said that this was a new package and then the automatic merge failed. I guess that this was because, being a new package, it needs to be vetted by a human?

> After having fixed the issue with the compat bounds, just retrigger the registration the same way you triggered it originally. Keep the same `version` in your `Project.toml`. That way, the registration will _update_ this PR, instead of creating a new one.

Thank you for your suggestions. I hope this time it is ok.

> You also have to edit your [comment](https://github.com/JuliaRegistries/General/pull/103950#issuecomment-2028780103) to contain `[noblock]`.

Ok. Thanks. I don't know what it does, but I guess I just need to add `[noblock]` somewhere in the text... 

[noblock]

goerz commented:
> In the section showing the automated checks

Those checks are kinda "technical logs" for the registry maintainer. As a "user" (someone registering a package), you're not really supposed to look at them: if there is an issue, the bot will communicate it in a [comment](https://github.com/JuliaRegistries/General/pull/103950#issuecomment-2028775163), which automatically updates when or if the issue is resolved (check the "edited" dropdown on that comment)

> apparently all had passed, but the last one

That last one is the important one, though.

> I guess that this was because, being a new package, it needs to be vetted by a human?

No, that was the check that flagged the `compat` issues. The "vetting by a human" only happens optionally. Although at this point there is a regular group of people keeping an eye on the registry, so most packages do get a very rudimentary vetting.

But mainly, the bot will just flag certain types of technical problems, like the missing `compat` bounds in this case. Most of these you can resolve yourself, and no maintainer has to manually interact with the PR.

Some issues that the bot flags, like "package name too short" can be overridden manually by a registry maintainer, but you'll have to get someone's attention (on Slack) if you need that.

Beyond that, the "human vetting" (by any member of the community) happens during the three-day registration window, and may flag issues beyond the automatic checks done by the bot. The most common issues are an unsuitable package name and missing documentation.

> I don't know what it does, but I guess I just need to add [noblock] somewhere in the text.

Yes, [like the bot explains](https://github.com/JuliaRegistries/General/pull/103950#issuecomment-2028775163), if there is any comment that does not contain `[noblock]`, the package will not be merged automatically at the end of the three-day period. This is how community members flag issues during the "manual vetting". (A registry maintainer can still override any blocking comment and merge manually, if there is a failure to reach consensus)

[noblock]
